modules/020-cni-cilium/docs/CONFIGURATION.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "The cni-flannel module: configuration"
+---
+
+The module is **automatically** enabled for the following cloud providers:
+- [OpenStack](../../modules/030-cloud-provider-openstack/)
+- [VMware vSphere](../../modules/030-cloud-provider-vsphere/)
```

Comment:
And what about Yandex? As i see it's also enabled by default there

Reply:
@yalosev, you've refactored the cni global hook, so this doesn't hold true anymore, correct?